app/views/requirement_manager/_form_fields.haml

```diff
@@ -1,5 +1,6 @@
 - help_text ||= ''
 - hide_inherited ||= false
+- on_unit ||= false
```

Comment:
This is fine, but only because the default is false.  We need to be careful with ||= and booleans.  A false passed in will evaluate to false.  If the default were true, then you'd never be able to pass in a false.  Might be safer to write `on_unit = false unless defined?(on_unit)`